packages/relay-service/tests/relay-service-e2e.test.js

```diff
@@ -1,9 +1,16 @@
 import {RelayService} from '../lib';
+import {dock} from '../lib/did/did-resolver';
 import {ALICE_KEY_PAIR_DOC, BOB_KEY_PAIR_DOC} from './mock-data';
 
 describe('Relay service', () => {
   const messageContent = `Test message ${Date.now()}`;
 
+  beforeAll(async () => {
+    await dock.init({
+      address: 'wss://knox-1.dock.io',
```

Comment:
should never connect to testnet node in tests/CI

Reply:
if this isnt an easy fix atm in wallet we can push it but it should use a local node always

Reply:
these should not run in the CI, was built for local testing